src/components/CardView/CardView.jsx

```diff
@@ -21,11 +21,11 @@ function CardView(props) {
   return (
     <div className='items-center bg-gray-50 rounded-lg shadow flex-col sm:flex lg:flex-row'>
       <Image
-        className='w-full rounded'
+        className='rounded w-[250px] h-[270px]'
         src={props.profile}
         alt='profile'
-        width={100}
-        height={100}
+        width={1920}
```

Comment:
The official Next.js documentation for the Image component states that "you should not use classes to override the width and height properties, as this will cause the image to be resized in the browser and lose quality."[1](https://nextjs.org/docs/api-reference/next/image)

Reply:
its already done right now